src/main/java/no/dcat/themes/service/CodesService.java

```diff
@@ -117,6 +156,14 @@ private static SkosCode extractLocation(Resource locationResource) {
                 result.addAll(extractLocationCodes(model, model.listResourcesWithProperty(GeoNames.officialName)));
                 result.addAll(extractLocationCodes(model, model.listResourcesWithProperty(RDF.type, AdmEnhet.NamedIndividual)));
 
+            } else if (type.equals(Types.openlicenses)) {
```

Comment:
kan du ikke bare legge dette som else og droppe det siste?

Reply:
Lagde en mapping som jeg bare vet gjelder for Licenses nå, gjelder det samme for alt her i bunn og grunn?

Reply:
Alle skal jo mappes om til SkosCode, eneste forskjellen er vel at noen OpenLicense har isReplacedBy